Meta.Tests.ps1

```diff
@@ -719,27 +719,47 @@ Describe 'Common Tests - Validate Markdown Files' -Tag 'Markdown' {
 
     if (Get-Command -Name 'npm' -ErrorAction SilentlyContinue)
     {
-        Write-Warning -Message "NPM is checking Gulp is installed. This may take a few moments."
-
-        $null = Start-Process `
-            -FilePath "npm" `
-            -ArgumentList @('install','--silent') `
-            -Wait `
-            -WorkingDirectory $PSScriptRoot `
-            -PassThru `
-            -NoNewWindow
-        $null = Start-Process `
-            -FilePath "npm" `
-            -ArgumentList @('install','-g','gulp','--silent') `
-            -Wait `
-            -WorkingDirectory $PSScriptRoot `
-            -PassThru `
-            -NoNewWindow
+        $npmAdditionalAdditionalParametersForStartProcess = @{
+            FilePath = 'npm'
+            ArgumentList = ''
+            WorkingDirectory = $PSScriptRoot
+            Wait = $true
+            WindowStyle = 'Hidden'
+        }
+
+        It 'Should not throw an error when installing dependencies' {
+            Write-Verbose -Message "NPM is checking Gulp is installed. This may take a few moments." -Verbose
+
+            {
+                <#
+                    gulp; gulp is a toolkit that helps you automate painful or time-consuming tasks in your development workflow.
+                    gulp must be installed globally to be able to be called through Start-Process
+                #>
+                $npmAdditionalAdditionalParametersForStartProcess['ArgumentList'] = 'install -g gulp'
```

Comment:
Can you separate the dependencies into their own `It` blocks rather than just the one big 'installing dependencies' `It` block? They seem more separation-friendly now.